src/main/java/org/kairosdb/datastore/cassandra/cache/CacheWarmingUpConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package org.kairosdb.datastore.cassandra.cache;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CacheWarmingUpConfiguration {
+    public static final Logger logger = LoggerFactory.getLogger(CacheWarmingUpConfiguration.class);
+
+    private final AtomicBoolean enabled = new AtomicBoolean(false);
+    private final AtomicInteger heatingIntervalMinutes = new AtomicInteger(120);
+    private final AtomicInteger rowIntervalInMinutes = new AtomicInteger(2);
+
+    public boolean isEnabled() {
+        return enabled.get();
+    }
+
+    public int getHeatingIntervalMinutes() {
+        return heatingIntervalMinutes.get();
+    }
+
+    public int getRowIntervalMinutes() {
+        return rowIntervalInMinutes.get();
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled.set(enabled);
+    }
+
+    public void setHeatingIntervalMinutes(final int newHeatingIntervalMinutes) {
+        if (newHeatingIntervalMinutes < 5) {
+            logger.warn(String.format("Discarding setting heatingIntervalMinutes to '%d' since it doesn't make sense", newHeatingIntervalMinutes));
```

Comment:
shouldn't this return early? same in other functions

Reply:
we actually tested it on staging with this values (5 and 1), so if it would return here, we couldn't test

Reply:
Good catch! Fixed